zwave_js/config.json

```
@@ -18,11 +18,13 @@
   "discovery": ["zwave_js"],
   "options": {
     "device": null,
-    "network_key": ""
+    "network_key": "",
+    "log_level": "info"
   },
   "schema": {
     "device": "device(subsystem=tty)",
     "network_key": "match(|[0-9a-fA-F]{32,32})",
+    "log_level": "list(silly|debug|verbose|http|info|warn|error)",
```

Make it optional? (note: does need handling to programmatically set a default when missing)

```suggestion
    "log_level": "list(silly|debug|verbose|http|info|warn|error)?",
```

---

I have tried setting the default `log_level` to an empty string but I can't save my configuration, even with the question mark. Do I need to have a blank option?

---

As far as I can tell, it's not possible to remove log_level entirely. I tested this in the zwavejs2mqtt hassio addon as well because I noticed it also makes this optional, and it didn't work there as well 🤷🏾‍♂️